services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -867,6 +865,13 @@ async def _clean_expired_uploads(self) -> None:
         async def _revert_file(
             conn: SAConnection, fmd: FileMetaDataAtDB
         ) -> FileMetaDataAtDB:
+            if is_valid_managed_multipart_upload(fmd.upload_id):
```

Comment:
why didn't we have this before? bug?

Reply:
no, it was handled by the other function that got deleted.